Effisense/sensor-dashboard
# Update container and sensor view (issue #28, #23, #35, #38)

Closes: #28, #23, #35, #38

## Checklist


## Changelog

- Update container view
- Update sensor view
- Populate default values based on existing data
- Redirect user if container or sensor id in URL doesn't exist
- Style 404 page
- Update database schema to reflect `onDelete` actions in our entities

## How to test

- Start Prisma Studio. Keep it open throughout the test
- Copy your container ID
- Navigate to `/containers/[id]/update`
- Observe that you get a loading state and default values from database populated in form
- Change some values
- Double check in Prisma Studio that the values were actually updated.
- Navigate to `/containers/something-nonexistent/update`
- Observe that you're redirected to `/404`, and it is styled
- Do the exact same routine for sensors, using `/sensors/[id]/update`
